python/2.0/offers-get.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
I'm confused a little my that directory naming: `1.0` + `2.0` it kinda looks like python versions.
How about `APIv1` + `APIv2` wouldn't that be more intuitive?

Reply:
And python path is mentioned in `README.md`
https://github.com/Currency-One/api-examples/blob/98d3c9283fd0a8055992f981e30d579cdcfeb463/README.md#L20

Reply:
thx